app/models/gc_organisation.js

```diff
@@ -0,0 +1,16 @@
+import Model from 'ember-data/model';
```

Comment:
@Darshan-Chauhan @abulsayyad123 @namrataukirde Just a general question. Would it make sense to have the models in goodcity-lib as well ? Since they are shared across apps.

Reply:
Agreed, I don't think it's urgent. We can look at refactoring when there's a bit of downtime

Reply:
Yes but I think we can move this model and route to `goodcity-lib` as it is part of this PR. 
For others we can create a ticket.

@abulsayyad123 , can you move this model to `goodcity-lib`?

Reply:
Sure, will move model and route to goodcity-lib.

Reply:
There, is problem to add `gc_organisation` model in goodcity-lib  right now because there is association for `designation` which exists in Stock but not in Browse app. And browse uses `order` instead of  `designation`. We can create seperate ticket for refactoring code.